doc/content/devices/troubleshooting/_index.md

```diff
@@ -225,3 +225,9 @@ This error indicates that your end device was added from the [Device Repository]
 The `COLLISION_PACKET` error occurs when downlink transmissions overlap - this happens when two or more packets overlap in time and use the same spreading factor, bandwith and frequency plan settings.
 
 To avoid packet collisions, users can enable server-side buffering of donwlink messages. If server-side buffering is enabled, the Gateway Server schedules the downlink message to be sent after the downlink message that was already queued but not sent yet, so that their transmissions don't overlap. To enable this, navigate to your gateway's **General settings** section, expand the **LoRaWAN** section and **Enable** the **Schedule downlink late** option by ticking the box. Note that this is a recommended setting for gateways that use [UDP packet forwarder]({{< ref "/gateways/concepts/udp" >}}).
+
+## I'm observing the `as.mqtt.connect.fail` event with error message `api key with id {api_key_id} not found` in {{% tts %}} Application live events. 
+
+The error message `api key with id {api_key_id} not found` could occur if the MQTT Integration API Key has been **deleted/removed** from {{% tts %}} Application in your TTS tenant. 
+
+We suggest you create a new MQTT API Key in the TTS Application and try subscribing to the TTS traffic with the new MQTT API Key. You can follow the [MQTT Integration guide]({{< ref "/integrations/mqtt" >}}) for detailed steps on subscribing and publishing the TTS traffic.
```

Comment:
Wrong place, this should be here https://www.thethingsindustries.com/docs/integrations/mqtt/#creating-an-api-key. You can just add a sentence such as:

If API key does not exist or has been removed, you will probably see the `as.mqtt.connect.fail` event with the `api key with id {api_key_id} not found` error in Live events.